spec.yaml

```diff
@@ -210,9 +210,10 @@ definitions:
     required:
       - shareWith
       - name
-      - providerId
+      - resourceId
```

Comment:
the providerID should still be part of the message

Reply:
Yes, maybe we can mark both providerId and resourceId as required (they would be repeating the same value) during a transition period, and then at the next major version we drop the deprecated providerId.